doc/neps/nep-0016-benchmark.py

```diff
@@ -8,7 +8,7 @@ class NotArray:
 class AttrArray:
     __array_implementer__ = True
 
-class ArrayBase(abc.ABC):
+class ArrayBase(abc.ABC):  # noqa: B024
```

Comment:
maybe you could sneak around this by using using `typing.Protocol` instead of `abc.ABC` 😈

Reply:
I wasn't aware of `typing.Protocol`, but I suspect that `# noqa: B024` will be easier to understand for future maintainers than inheriting a random subclass of `abc.ABC` without apparent reason.